shaders/src/shading_lit.fs

```diff
@@ -4,7 +4,7 @@
 
 float computeDiffuseAlpha(float a) {
 #if defined(BLEND_MODE_TRANSPARENT) || defined(BLEND_MODE_FADE) || defined(BLEND_MODE_MASKED)
-    return a;
+    return (frameUniforms.needsAlphaChannel == 1.0) ? min(1.0, a + 0.125) : a;
```

Comment:
What were the reasons to change `colorGrading.mat`? That broke "Simple Transparent", and IMHO this hack is harder to explain and justify than what we had there.